### ISSUE TITLE
Make today highlight color more visible

### ISSUE THREAD

**Issue by abyssox** [libasys/calendarplus#128]
Please make the current today highlight color more visible.
Like #FFC2D3 or similar:
![2015_08_27-08_29_32](https://cloud.githubusercontent.com/assets/4539953/9513961/0ef54f60-4c96-11e5-9a4d-f38276a32f85.png)
##

**Comment by deMattin** [comment]
It's a problem of monitor.
I also have a very view angle dependend monitor and there also the actual day color is mostly not viewable.
I don't like such a very dark red (pink) color but maybe the border of actual day can get additional "full red without transparency" like the color of actual time marker in week view for monitors not displaying colors well:
`td.fc-today {border-color: red;}`

I tried it, but the right and bottom border are set to "0" and so the border is incomplete.
![image](https://cloud.githubusercontent.com/assets/5565503/9515184/faa8b43e-4c9f-11e5-9cc7-6ae858afe5e5.png)

But it doesn't look very bad like this ... ;)

**Comment by deMattin** [comment]
It's not very simple with such a border.
If using `fc-state-highlite`, which would solve it for all views, only another color does make sense.
So "border" probably is not a good solution and it would be better, to find a color, that is also useable on "bad monitors".
@jancborchardt?

**Comment by deMattin** [comment]
What's about using the border color of days in calendar also as actual day marker?
`.fc-state-highlight {background: #E7E7DE;}`

![image](https://cloud.githubusercontent.com/assets/5565503/9515822/683f5d50-4ca4-11e5-9719-1ad1bfcee39e.png)

I don't know, how it would work with weekends color, but as borders are seen there, the contrast seems to be sufficient.

**Comment by libasys** [comment]
i think this is a feeling somebody prefer this color somebody else another color! But every user can change the color for the highlighted day in the file css/3rdparty/fullcalendar.css
https://github.com/libasys/calendarplus/blob/master/css/3rdparty/fullcalendar.css#L144-L146

**Comment by deMattin** [comment]
@libasys: You would be right, if it would be only a matter of taste.
But the highlighting of actual day isn't viewable at all on some monitors in most usecases/viewangles!

If someone doesn't like the color, user can simply change it - but default color should work also on "not so well" (viewangle dependend) monitors what actually isn't like that.

**Comment by abyssox** [comment]
@libasys i agree with you completly, it's a personal preference but as @deMattin mentioned on some monitors it really isn't viewable.

**Comment by libasys** [comment]
@calis2002 and @deMattin i have added the stylesheet definition for highlighting today column!
https://github.com/libasys/calendarplus/blob/bc43b963e47818f94d35aecbfe78a9002e352719/css/style.css#L2099%20-%20L2115
play around and then say what is the best color for todays!

**Comment by deMattin** [comment]
@jancborchardt?

**Comment by jancborchardt** [comment]
How about making the date (number of day) of the current day bold to support the highlight? That will look much better than a color highlight and work even in bad lighting conditions.
